src/common/getters.ts

```diff
@@ -212,6 +212,235 @@ export function getPurposes(vc: DatasetWithId): string[] {
   return purposes;
 }
 
+/**
+ * Reads the custom string value with the provided name in the consent section of the provided Access Credential.
+ *
+ * @example
+ * ```
+ * const accessRequest = await issueAccessRequest({...}, {
+ *  ...,
+ *  customFields: new Set([
+ *     {
+ *       key: new URL("https://example.org/ns/customString"),
+ *       value: "custom value",
+ *     },
+ *   ]),
+ * });
+ * // s is "custom value"
+ * const s = getCustomString(accessRequest, new URL("https://example.org/ns/customString"));
+ * ```
+ *
+ * @param accessCredential The Access Credential (Access Grant or Access request)
```

Comment:
For typedocs, does the param name need to match the method, as in `vc`. If so, see all below as well

Reply:
Fixed in ace91b1c37a503e59c354e0c90bc22e2c5697666